index.html

```diff
@@ -338,7 +353,8 @@
           </p>
         </div>
         <p>
-          Each <a>DepthData</a> interface is associated with a
+          Each <code><a>ImageData</a></code> interface that <a>represent depth data</a>
```

Comment:
Fixed with https://github.com/anssiko/mediacapture-depth/commit/b69631c7856262dbe5e9b21f48ed79c1483f0223

(We should keep definitions (&lt;dfn&gt;) singular, and tweak the language around the definition to fit the singular form. Otherwise, autolinking to definitions does not work.)